platforms/documentation/docs/src/docs/release/notes-template.md

```diff
@@ -10,6 +10,8 @@ Include only their name, impactful features should be called out separately belo
 -->
 We would like to thank the following community members for their contributions to this release of Gradle:
 
+Be sure to check out the [Public Roadmap](https://blog.gradle.org/roadmap-announcement) for insight into what's planned for future releases.
```

Comment:
This line should be added in the `notes.md` file as well, since it was created based on the template without it.